app/scripts/views/form.js

```diff
@@ -199,6 +199,10 @@ define(function (require, exports, module) {
 
     _submitForm: notifyDelayedRequest(showButtonProgressIndicator(function () {
       var self = this;
+      // sets all password fields to type=password
+      self.$el.find('.password').each(function (i, el) {
```

Comment:
This seems like it should belong in the [password mixin](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/mixins/password-mixin.js) to keep password element related logic in one place.